servers/quarkus-server/src/test/java/org/projectnessie/server/error/TestNessieError.java

```diff
@@ -134,6 +135,18 @@ void blankParameterQueryGet() {
                     "Bad Request (HTTP/400): blankParameterQueryGet.hash: must not be blank"));
   }
 
+  @Test
```

Comment:
Do we need this test (and the change in ErrorTestService)? Not sure what's different to `AbstractResteasyTest.testGetUnknownContentType()`

Reply:
I'd be fine with keeping either one of these and deleting the other.

I wrote this test first (with the associated change in `ErrorTestService`), then wrote the `AbstractRestEasyTest` case second.

I tentatively left them both in because they looked moderately cheap to execute -- locally ~10ms for the `TestNessieError` case and ~90ms for the other case on in-memory, though surely slower on other stores -- and I wasn't sure whether the difference in backends might eventually come in handy by revealing some interaction the `ErrorTestService` stub couldn't induce.  So, my response to that uncertainty was to err on the side of redundant coverage, but I really don't have a strong opinion here.

Incidentally, aside from deleting this, I'd be happy to add test coverage elsewhere, if you have suggestions.

Reply:
Hm - I don't have a strong opinion here. Fine to keep both to validate it in a slightly different way in Quarkus.